.circleci/config.yml

```diff
@@ -539,9 +539,6 @@ commands:
       reattach-enabled:
         type: boolean
         default: false
-      webhooks-base-url:
```

Comment:
rebase issue?

Reply:
weird, we haven't figured out what caused it

Reply:
we figured it out, refreshing was the answer.